server/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "bcrypt": "^5.0.0",
     "express": "^4.17.1",
+    "mariadb": "^2.5.2",
     "morgan": "^1.10.0",
     "mysql2": "^2.2.5",
```

Comment:
You should uninstall the mysql2 package as we do not use it anymore. In rest, it's ok

Reply:
did we switch to `mariadb`? I did it only locally, since I had a bug.